DashWallet/Sources/UI/CrowdNode/New Account/AccountCreatingController.swift

```diff
@@ -53,10 +53,7 @@ final class AccountCreatingController: UIViewController {
 
 extension AccountCreatingController {
     private func configureHierarchy() {
-        let appearance = UINavigationBarAppearance()
-        appearance.configureWithTransparentBackground()
-        navigationItem.standardAppearance = appearance
-        navigationItem.scrollEdgeAppearance = appearance
+        view.backgroundColor = .dw_secondaryBackground()
```

Comment:
This creates problems with UI. By design, we have a white background with a gray button here: https://www.figma.com/file/84tUBQA9DYU1svyQx8tnbk/Dash-Wallet%E3%83%BBIn-process%E3%83%BBAndroid?node-id=13128%3A14702&t=z4LVy8U1iXqA1XMw-0

<img src="https://user-images.githubusercontent.com/7733989/218631144-b26ea454-95cb-4460-a7e5-835dd2262fc0.jpeg" width="300" />

Reply:
Fixed: Used the same we use for blocks like 'Primary Dash address' for the button